drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -24,12 +24,133 @@
 
 #include "fsl_xcvr.h"
 
+/*
+ * KW41Z M0 has Micro Trace Buffer support (MTB). The MTB is a hardware
+ * program flow tracing that traces the branch points of execution.
+ * Where this proves useful is when a hard fault occurs in the M0 you
+ * can have a breakpoint in the __reserved function (reset) and when
+ * the breakpoint happens look at the contents of the mtb buffer at
+ * the addresses traced.
+ *
+ * In gdb use: 'info line *<addr>'
+ *
+ */
+/*#define KW41Z_DEBUG_MTB_ENABLED   1*/
+
+#if KW41Z_DEBUG_MTB_ENABLED
```

Comment:
lets delete the trace stuff for now.

Reply:
sigh... it is useful for debug... but if you really want it out I'll pull it to a JIRA ticket notes

Reply:
Please just separate it to a different pull request. It still won't be mergeable, but will be a good start even for someone else to pick up and refactor even if you don't do that (everyone hopes you will ;-)).

Reply:
I know its useful, but its kinda like having code like:

#if 0
printk("this is my debug foo")
#endif  

I'm all for supporting a feature like this, and its useful so I think Paul's comment of a separate PR with it would be good, that we can than use to discuss and move to a more generic solution.  Just don't want to clutter or hold up this PR on trace support/issues.